blockchain/tx_list.go

```diff
@@ -380,6 +392,16 @@ func (l *txList) Filter(sender common.Address, pool *TxPool) (types.Transactions
 				return senderBalance.Cmp(tx.Value()) < 0 || feePayerBalance.Cmp(tx.Fee()) < 0
 			}
 		}
+
+		// balance check for module transaction
+		for _, module := range pool.modules {
+			if module.IsModuleTx(pool, tx) {
+				if checkBalance := module.GetCheckBalance(); checkBalance != nil {
+					return checkBalance(pool, tx) != nil
```

Comment:
Only the first module's GetCheckBalance is executed, unlike in `validateTx` where the function is executed for all modules.

Reply:
https://github.com/ulbqb/kaia/blob/949f09bb4032b2bbf179a65aa2dba02cb3f26ede/blockchain/tx_pool.go#L797

`validateTx` is the same as well.

Reply:
I missed it, thanks